### ISSUE TITLE
Some problems with wm_with_flattened_interventions_metadata.yml

### ISSUE THREAD

**Issue by kwalcock** [WorldModelers/Ontologies#44]
I noticed that tests would complain about some of the spaces in this file.  Also there isn't a problem yet, but I think somebody standardized on polarity being an integer not to long ago and this file uses both 1 and 1.0.  It should at least pick.  I may add a test for this.

```
Spaces: ArrayBuffer(wm, concept, causal_factor, interventions, Information Services)
Spaces: ArrayBuffer(wm, concept, causal_factor, interventions, Teaching, Training and Public Awareness)
Spaces: ArrayBuffer(wm, concept, causal_factor, interventions, Teaching, Training and Public Awareness, Training of Farmers on Weed Control)
Spaces: ArrayBuffer(wm, concept, causal_factor, interventions, Teaching, Training and Public Awareness, Training of farmers on crop spacing)
Spaces: ArrayBuffer(wm, concept, causal_factor, interventions, Teaching, Training and Public Awareness, Training of farmers on planting times)
Spaces: ArrayBuffer(wm, concept, causal_factor, interventions, Building Construction)
Spaces: ArrayBuffer(wm, concept, causal_factor, interventions, provision_of_goods_and_services, Fertilizer Subsidy)
```

**Comment by bgyori** [comment]
Thanks! I was involved in both of those choices i.e., standardizing to names with underscores instead of spaces, and integer instead of float polarities. Is one of these the root cause behind https://github.com/clulab/eidos/issues/717?

**Comment by kwalcock** [comment]
I don't think so, but I'm still looking.

**Comment by BeckySharp** [comment]
the "choices" in the new file aren't on purpose (the spaces in names were
just me keeping whatever Kimetrica had used, and the same with the
polarities), sorry!
I can try to change today if it's a problem

**Comment by bgyori** [comment]
Yes, for our use it would be nice to standardize to underscores in names and integer polarities unless there are any objections.

**Comment by BeckySharp** [comment]
ok, cool will make the changes and try to post very soon (my internet isn't
being my bestest friend at the moment but I think I can figure it out)

**Comment by BeckySharp** [comment]
addressed with #45
